.github/CODEOWNERS

```diff
@@ -143,10 +143,12 @@
 
 # stdlib
 # TODO: /stdlib/
+/stdlib/public/core/                      @stephentyrone
 /stdlib/public/Backtracing/               @al45tair
 /stdlib/public/Concurrency/               @ktoso @kavon
```

Comment:
Please insert new entries in lexicographic case-sensitive order (capitals first). But I don't mind if this gets corrected with the upcoming update.